backend/models.py

```diff
@@ -78,6 +78,49 @@ class POIU(db.Model, AsDictMixin):
 
     def __repr__(self):
         return f"<POIU {self.id}>"
+    
+class playerShiftTrack(db.Model, AsDictMixin):
+
+    id = db.Column(db.Float, primary_key=True)
+
+    game_id = db.Column(db.Integer, nullable=True)
+    shift_period = db.Column(db.Integer, nullable=True)
+
+    player_id = db.Column(db.Integer, nullable=True)
+    team_id = db.Column(db.Integer, nullable=True)
+    shift_number = db.Column(db.Integer, nullable=True)
+    
+    start_shift_number = db.Column(db.Float, nullable=True)
+    end_shift_number = db.Column(db.Float, nullable=True)
+    duration_number = db.Column(db.Float, nullable=True)
+
+    def __repr__(self):
+        return f"<PlayerShiftTrack {self.id}>"
+
+class gameShiftTrack(db.Model, AsDictMixin):
```

Comment:
Please change this to Pascal Case.
```suggestion
class GameShiftTrack(db.Model, AsDictMixin):
```